frontend/src/services/transactions/request-manager.ts

```diff
@@ -27,7 +28,7 @@ export async function sendRequestTransaction(
   requestState: Ref<RequestState>,
 ): Promise<Request> {
   if (!request.fee) {
-    request.fee = await getRequestFee(signer, request);
+    request.fee = await getRequestFee(request);
```

Comment:
This should be somehow the requestManagerAddress and not the full request object! I doubt that this condition has ever been verified before so we didn't see the error that will happen in this case :D

Reply:
Yes, this is a type error and in another branch with more advanced linting I removed that part properly.
I feel like I can just close this conversation. 😉